packages/sources/coinmetrics/src/endpoint/lwba-ws.ts

```diff
@@ -1,27 +1,15 @@
 import {
```

Comment:
Similar to the `-router` comment if you agree, we might not need `-ws` here on the endpoint anymore

Reply:
I think in this case this was my mistake since lwba is `ws` only  and suffix is not needed (Regardless if i agree about `router` or not). Same in the transport